src/main/java/jp/co/soramitsu/sora/deauth/DAuthHeaders.java

```diff
@@ -0,0 +1,10 @@
+package jp.co.soramitsu.sora.deauth;
+
+public class DAuthHeaders {
```

Comment:
add final keyword for this class and private constructor to secure this class from extending